### ISSUE TITLE
FYI org-journal support has been merged into the org layer in the develop branch of spacemacs

### ISSUE THREAD

**Issue by nickanderson** [borgnix/spacemacs-journal#5]
If your using the develop branch in spacemacs you can enable org-journal and configure it:

Docs: https://github.com/syl20bnr/spacemacs/tree/develop/layers/%2Bemacs/org#org-journal-support

**Comment by ruibinx** [comment]
Thanks. I will consider updating README when it come to release.

**Comment by willtm** [comment]
Hello, I like what you've done with `org-journal` @borgnix.  But searching the journal does not work for me.

Do you have any guesses for how long it will take for the `org-journal` changes that are [described in the develop branch]( https://github.com/syl20bnr/spacemacs/tree/develop/layers/%2Bemacs/org#org-journal-support) cited above?  It appears that searching works on the develop branch.

**Comment by ruibinx** [comment]
@willtm Have you tried updating org-journal package? I noticed that the org-journal package recently released a fix which might help.

**Comment by willtm** [comment]
@borgnix thank you for responding so quickly!  I am now able to search after installing the latest version of `org-journal` ( Version: 20170928.231) and `spacemacs-journal`.  And is it turns out, I made a thoughtless error by failing to install `org-journal` in my earlier attempt and just installing `spacemacs-journal`.

**Comment by ruibinx** [comment]
@willtm Glad to know it works.

**Comment by x61** [comment]
First of all, I was unable to get this to work. I clone the branch and initialized, per the instruction; but it won't work. It kept on saying `SPC a j is undefined`.  Does the native org-journal in spacemacs use the same keybindings as does this one? With the native setup, I can create a new entry with `C-c C-j` but I would like to use the keybindings you defined here. Any suggestions?

**Comment by ghost** [comment]
No it doesn't  built in org-journal  is   SPC-a-o-j   or the mnemonic  Leader  application org journal

**Comment by nickanderson** [comment]
https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Bemacs/org/README.org#org-journal

**Comment by x61** [comment]
Is it possible to use these same keybindings while on `develop` branch? I want to use the same keybindings and but have not been successful while on `develop` branch.

**Comment by nickanderson** [comment]
Yes, those are what is currently on the `develop` branch. The spacemacs readme says .300 is "almost out" https://github.com/syl20bnr/spacemacs/blob/develop/core/info/release-notes/0.300.txt

**Comment by ruibinx** [comment]
@x61 you don't need this layer any more if you are on develop branch, and please read https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Bemacs/org/README.org#org-journal since the keybindings are slightly different.

**Comment by x61** [comment]
@borgnix, is it possible to have the same keybindings even when you `develop` branch? If so, how do I do it? Or is it possible to add the layer even when on develop branch?   Muscle memory.
